benchmarks/php_php-fuzz-execute/Dockerfile

```diff
@@ -17,7 +17,14 @@
 FROM gcr.io/oss-fuzz-base/base-builder@sha256:fb1a9a49752c9e504687448d1f1a048ec1e062e2e40f7e8a23e86b63ff3dad7c
 RUN apt-get update && \
     apt-get install -y autoconf automake libtool bison re2c pkg-config
-RUN git clone --depth 1 --branch master https://github.com/php/php-src.git php-src
-RUN git clone https://github.com/kkos/oniguruma.git php-src/oniguruma
+RUN git clone \
```

Comment:
The correct way to pin these benchmarks is here: https://github.com/google/fuzzbench/blob/master/benchmarks/php_php-fuzz-execute/benchmark.yaml#L1

If none of the bug based benchmarks upgrade (which is possible) I'm happy to junk them.